src/pages/registration/contributors_tab/components/SelectAffiliationRadioButton.tsx

```diff
@@ -19,6 +20,7 @@ export const SelectAffiliationRadioButton = ({
   setSelectedPerson,
   affiliationIsSelected,
   personIsSelected,
+  disabled = false,
```

Comment:
Var borti en lignede sak på `CategoryChip.tsx` for noen dager siden. Der var det en `disabledText`-prop som kunne sendes inn som betydde at en chip var disabled, men kom også et behov for å kunne sette disabled uten å ha noen `disabledText`. For å unngå at de var i usynk endte landa jeg på å sette `disabled = !!category.disableText` som default. Dermed vet jeg at `disabled` alltid får riktig verdi, og den kan også settes implisitt av en annen prop, og jeg slapp en litt rar OR-sjekk på disabled for Chip/Button, siden man burde anta at `disabled`-prop alltid er fasit og ikke trenger å sjekke mot noe annet samtidig

Vet ikke om det passer inn her da, men kan være noe å se på

```suggestion
  disabled = !personIsSelected,
```